src/dataExplorer/components/FieldSelector.tsx

```diff
@@ -84,10 +91,10 @@ const FieldSelector: FC = () => {
     ))
   }
 
-  const handleLoadMore = () => {
+  const handleLoadMore = useCallback(() => {
     const newIndex = fieldsToShow.length + LOAD_MORE_LIMIT
     setFieldsToShow(fields.slice(0, newIndex))
-  }
+  }, [fieldsToShow])
```

Comment:
you gotta add `fields` and `setFieldsToShow` here as well!

Reply:
oh! let me try!